### DISCUSSION TITLE
Padding-Left on :Lspsaga symbols in winbar

### DISCUSSION THREAD

**Discussion by MannyFay** [nvimdev/lspsaga.nvim#1098] [Q&A]
Is there a way to add a little bit of padding-left on :Lspsaga symbols in winbar on the whole line?

**Comment by glepnir** [comment]
add some space on `symbol_win_winbar.separator` like `  >  `

**Comment by MannyFay** [comment]
Thank's for the reply :)
But that I've done already:
![Screenshot 2023-06-09 at 12 49 03](https://github.com/nvimdev/lspsaga.nvim/assets/45234439/c7b98308-86b5-4a70-ab6f-a1869157329b)

I mean, how to add more space on the very left. In the screenshot, between vertical gray buffer line and the folder symbol.

**Comment by glepnir** [comment]
oh you mean first fold name left . currently not support . but I will add this support in next version

**Comment by MannyFay** [comment]
![Screenshot 2023-06-09 at 12 58 07](https://github.com/nvimdev/lspsaga.nvim/assets/45234439/15e95585-c6b5-44cf-8184-5899c4542a6d)
The space between the folder-symbol and the vertical gray buffer line I'd like to change.
